lib/chef/provider/windows_task.rb

```diff
@@ -601,6 +603,17 @@ def run_schtasks(task_action, options = {})
         end
         # rubocop:enable Style/StringLiteralsInInterpolation
 
+        def task_user
```

Comment:
Can you toss the comment on the top of the method here so it gets properly documented by yard. A yard return tag would also be useful for the web docs. https://gist.github.com/chetan/1827484

Reply:
Done.